custom/icds/management/commands/migrate_pse_attendance.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import print_function
+
+from django.core.management.base import BaseCommand
+
+from django.db import connections
+
+
+new_table = 'config_report_icds-cas_static-child_cases_monthly_v2_198ccc06'
+old_table = 'config_report_icds-cas_static-child_cases_monthly_tabl_551fd064'
+
+migration_query = """
+UPDATE {new_table} B
+SET
+B.pse_daily_attendance = A.pse_days_attended,
+B.pse_daily_attendance_male = CASE WHEN B.sex = 'M' THEN A.pse_days_attended ELSE NULL END,
+B.pse_daily_attendance_female = CASE WHEN B.sex = 'F' THEN A.pse_days_attended ELSE NULL END
```

Comment:
just realized these aliases for `B.` should not be on the left side of the equals sign here (my mistake). the aliases on the right side are fine though.